PocketKit/Sources/PocketKit/MyList/EmptyStates/EmptyStateView.swift

```diff
@@ -1,36 +1,36 @@
 import SwiftUI
 import Textile
 
-open class SwiftUICollectionViewCell<Content>: UICollectionViewCell where Content: View {
-    private(set) var host: UIHostingController<Content>?
+open class SwiftUICollectionViewCell<T>: UICollectionViewCell where T: View {
+    private(set) var hosting: UIHostingController<T>?
```

Comment:
This was needed because once you import Listen, this name somehow conflicts. Dont ask me why, cause I have no idea.